ai-service/src/ai_service/embeddings/transformer.py

```
@@ -25,32 +28,36 @@ def _get_device() -> str:
     return "cpu"
 
 
-@lru_cache(maxsize=1)
-def get_model(
-    trust_remote_code: bool = False,  # Might be required for some code models
-) -> SentenceTransformer:
+def initialize_model(trust_remote_code: bool = False) -> None:
```

Ruff (0.12.2)

Boolean-typed positional argument in function definition

(FBT001)